### PR TITLE
torch.nn.functional.interpolate frontend 

### PR DESCRIPTION
**Pull request by shirsho-12**
I've written code for the frontend, but I am not sure if the exception-handling code should be here. Other than that, to my knowledge ivy.interpolate is being worked on for the backend.

### CODE REVIEW

**Comment by shirsho-12** [comment]
close #7689

File: ivy_tests/test_ivy/test_frontends/test_torch/test_vision_functions.py [hunk 1]
```diff
@@ -6,6 +6,22 @@
 from ivy_tests.test_ivy.helpers import handle_frontend_test
 
 
+# interpolation
+@handle_frontend_test(
```
Comment (simonetgordon) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_vision_functions.py:10]: Hi there, can you please try to write a test function for this? It will help you to write the function too!

**Comment by simonetgordon** [comment]
@shirsho-12 Are you still working on this issue?

**Comment by shirsho-12** [comment]
Ah apologies, i haven't gotten to it in some time. I'll finish it before new years. If not feel free to close this.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.
